### PR TITLE
Make the schema version match the current version

### PR DESCRIPTION
**Pull request by andrewpollock**

### CODE REVIEW

**Review by oliverchang — APPROVED** [review]

File: osv/models.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 from . import sources
 from . import vulnerability_pb2
 
-SCHEMA_VERSION = '1.3.0'
+SCHEMA_VERSION = '1.3.1'
```
Comment (another-rex) [inline osv/models.py:33]:
```suggestion
SCHEMA_VERSION = '1.4.0'
```

Comment (another-rex) [reply osv/models.py:33]: Updated again to 1.4.0

Comment (oliverchang) [reply osv/models.py:33]: We don't actually support 1.4.0 properly yet. I created https://github.com/google/osv.dev/issues/1093 to track the needed changes.

**Comment by michaelkedar** [comment]
This was done in #1115

**Pull request closed without merging**